sdk/deviceupdate/azure-iot-deviceupdate/CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## 1.0.0-beta.2 (Unreleased)
 
+### Features Added
```

Comment:
AFAIK, you need to remove the other empty sections for release (so only Breaking Changes remains).

You might also add a Dependency Updates like this https://github.com/weidongxu-microsoft/azure-sdk-for-java/blob/03f23db5b5dc2b006f0d33f1c9f9140813e005c7/sdk/webpubsub/azure-messaging-webpubsub/CHANGELOG.md
You can ignore the "from" part, since the lib is ancient.